### PR TITLE
Support variables in url

### PR DESCRIPTION
**Pull request by woutervb**
Handle variables inside url definitions for mocks
    
When url's are defined with <variable> definitions in the url, make sure
that these are transformed into regexp urls.
    
Ie change an url from:

    http://localhost:8005/invites/<package_type>/<package_id>/pending
    
To:
    
    http://localhost:8005/invites/(?P<package_type>\S+)/(?P<package_id>\S+)/pending
    
Which will allow responses to match any call that had a random
package_type and package_id

### CODE REVIEW

File: acceptable/mocks.py [hunk 1]
```diff
@@ -278,6 +278,11 @@ class Endpoint(object):
     def __init__(self, base_url, service_name, endpoint_spec, response_callback=None):
         if isinstance(endpoint_spec.location, str):
             self._url = urljoin(base_url, endpoint_spec.location)
+            if self._url.find('<') >0 and self._url.find('<') >0:
```
Comment (tong-canonical) [inline acceptable/mocks.py:281]: space between > and 0?

File: acceptable/mocks.py [hunk 2]
```diff
@@ -278,6 +278,11 @@ class Endpoint(object):
...
+                # we know that there are variable references in the url, so
+                # let's change the url in a regexp
+                self._url=self._url.replace('<', '(?P<').replace('>', '>\S+)').replace('int:', '')
```
Comment (tong-canonical) [inline acceptable/mocks.py:284]: space before and after =

**Review by tong-canonical — APPROVED** [review]
+1, now that we support this in acceptable, we may need to update the workaround that I implemented in some of the test cases for Collaborators API. We will cross that bridge when we get there. :)

File: acceptable/mocks.py (hunk 2 above)
Comment (woutervb) [reply acceptable/mocks.py]: Fixed

**Pull request merged**